po/nl.po

```diff
@@ -228,7 +228,11 @@ msgstr "%s. Stoppen annuleert alle overgebleven transacties"
 msgid "Close"
 msgstr "Sluiten"
 
-#: ../src/MainWindow.vala:179
+#:../src/MainWindow.vala: 197
```

Comment:
I'm not sure if this is syntactically relevant, but the other lines don't have a space between the colon and the line number (and there *is* a space after the first colon at the start of the line, which is missing here). You might want to move the space even if just for consistency's sake :)